tests/cupy_tests/fft_tests/test_fft.py

```diff
@@ -50,6 +50,42 @@ def test_func(self, *args, **kw):
     return decorator
 
 
+def multi_gpu_config(gpu_configs=None):
+    """Decorator for parameterized tests with different GPU configurations.
+
+    Args:
+        gpu_configs(list of list): The GPUs to test.
+
+    .. notes:
+        The decorated tests are skipped if no or only one GPU is available.
+    """
+    def decorator(impl):
+        @functools.wraps(impl)
+        def test_func(self, *args, **kw):
+            use_multi_gpus = config.use_multi_gpus
+            _devices = config._devices
+
+            try:
+                for gpus in gpu_configs:
+                    try:
+                        nGPUs = len(gpus)
+                        if nGPUs < 2:
+                            raise ValueError('Must use at least two gpus')
```

Comment:
Please use `assert len(gpus) >= 2` because `testing.numpy.cupy_*(..., accept_error=ValueError, ...)` cannot distinguish between this `ValueError` and a `ValueError` raised in a CuPy's function.

Reply:
Thank you. I didn't consider this. I fixed it.